src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor({ health = 100, name }) {
```

Comment:
The constructor of the Animal class should take a `name` parameter, not an object. The current implementation will cause `name` to be `undefined` since you are destructuring a non-existent object. Change the constructor signature to `constructor(name, health = 100)`.